frontend/packages/topology/src/components/page/TopologyPageToolbar.tsx

```diff
@@ -25,8 +27,8 @@ const TopologyPageToolbar: React.FC<TopologyPageToolbarProps> = observer(
       <>
         {showGraphView ? (
           <Popover
-            aria-label="Shortcuts"
-            bodyContent={TopologyShortcuts}
+            aria-label={t('topology~Shortcuts')}
+            bodyContent={getTopologyShortcuts(t)}
```

Comment:
The API accepts a `ReactNode`. `TopologyShortcuts` can make use of `useTranslation`
```suggestion
            bodyContent={<TopologyShortcuts />}
```